wechat/handlers.py

```diff
@@ -65,3 +75,170 @@ def check(self):
 
     def handle(self):
         return self.reply_text(self.get_message('book_empty'))
+
+
+class BookWhatHandler(WeChatHandler):
+
+    def check(self):
+        return self.is_text('抢啥') or self.is_event_click(self.view.event_keys['book_what'])
+
+    def handle(self):
+        lib = WeChatLib(WECHAT_TOKEN, WECHAT_APPID, WECHAT_SECRET)
+        wechat_menu = lib.get_wechat_menu()
+        if len(wechat_menu) < 2:
+            return self.reply_text('暂时没有可抢票的活动')
+        elif len(wechat_menu[1]['sub_button']) == 0:
```

Comment:
the capacity of menu is limited, so there may be some activity avilable but not in the menue. so the filter condition should not be 'activity in the menu' but 'activity avilable'